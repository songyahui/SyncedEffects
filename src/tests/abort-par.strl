module abortpar:

input I;
output O;

signal L in
	       abort
			loop
				emit O;
				pause;
			end
	       when L
	||
		[ await I;
		  emit L; ]
end signal

end module