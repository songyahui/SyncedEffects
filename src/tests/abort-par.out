abortpar> ;
--- Output: O
abortpar> ;
--- Output: O
abortpar> I;
--- Output:
abortpar> ;
--- Output:
abortpar> ;
--- Output:
